app/views/exam_templates/assign_errors.html.erb

```diff
@@ -0,0 +1,279 @@
+<% content_for :head do %>
+    <%= stylesheet_link_tag('grader') %>
+    <%= stylesheet_link_tag('pdfjs/viewer') %>
+    <%= stylesheet_link_tag('context_menu') %>
+
+    <%= javascript_include_tag 'Results/file_selector',
+                               'panes',
+                               'pdfjs',
+                               'testresults_config',
+                               'Results/context_menu' %>
+
+    <%= javascript_tag do -%>
+        PDFJS.workerSrc = "<%= @host %>/assets/pdfjs_lib/pdf.worker.js";
+    <% end -%>
+<% end %>
+
+<% # SourceCodeGlower, ImageAnnotations, and SyntaxHighlighter brushes %>
```

Comment:
No it wasn't on Assign scans, but unfortunately for me to get the PDF viewer toolbar/rotation working it required a AnnotationManager object to be defined. This was a temporary workaround.